src/Orchard/ContentManagement/Utilities/LazyField.cs

```diff
@@ -11,7 +11,7 @@ public class LazyField<T> {
             set { SetValue(value); }
         }
 
-        public void Loader(Func<T, T> loader) {
+        public void Loader(Func<T> loader) {

```

Comment:
I guess that's a good idea. However, @sebastienros has the final word in this matter.